python_scripts/03_categorical_pipeline_sol_02.py

```diff
@@ -153,3 +153,20 @@
 # The main take away message is that arbitrary integer coding of
 # categories is perfectly fine for `HistGradientBoostingClassifier`
 # and yields fast training times.
+
+# %% [markdown]
```

Comment:
After the work I did on generating the exercise from the solutions in #408, you need to use `# %% [markdown] tags=["solution"]`, see https://github.com/INRIA/scikit-learn-mooc/blob/master/CONTRIBUTING.md#generating-exercises-from-the-solutions for more details.

Also make sure that doing `make exercises` is not modifying your particular exercise (It will modify other exercises which we need to fix in #418